bot/plugins/help.py

```diff
@@ -0,0 +1,57 @@
+import crescent
+import hikari
+
+from bot.app import Plugin
+from bot.constants import EMBED_COLOR
+
+plugin = Plugin()
+
+HELP_EMBEDS = [
+    hikari.Embed(
+        description=(
+            "Hi! My name is io, and my job is to run code."
+            "\n I can run any code inside of code blocks:"
+            "\n\\`\\`\\`<language-name>"
+            "\n<your code here>"
+            "\n\\`\\`\\`"
```

Comment:
You can use zero-width spaces between the back-ticks instead. Then surround the whole thing in a code block. The result is this:
![64341](https://github.com/CircuitSacul/io/assets/55721663/210fac05-64f4-4700-addd-1ea876c1d588)
![33248](https://github.com/CircuitSacul/io/assets/55721663/b91595fc-ebfd-4d99-b568-ede9b0d81893)

Reply:
If you think the non-code-block version is more readable I'm willing to change my mind. Just thought I'd mention that this *is* possible too.

Reply:
i think the code block looks better